routes/offers.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const jwt = require("jsonwebtoken");
+const bodyParser = require("body-parser");
+require("dotenv").config();
+
+const { getOffers, getOffer, createOffer } = require("../lib/offers");
+
+const router = express.Router();
+router.use(bodyParser.json());
+
+function createOffersRouter(database) {
+  router.get("/test", (request, response) => {
+    response.send("We're on offer route!");
+  });
+
+  // GET OFFERS
+  router.get("/", async (request, response) => {
+    try {
+      const { authToken } = request.cookies;
+      const { username } = jwt.verify(authToken, process.env.JWT_SECRET);
+      console.log(`Allow access to ${username}`);
+
+      const offers = await getOffers(database);
+      if (!offers) {
+        response.status(404).send("No offers available.");
+        return;
+      }
+
+      response.status(200).send(offers);
+    } catch (error) {
+      console.error(error);
+      console.error("Something went wrong 😑", error);
```

Comment:
Please put the Smileys in a <span>>/span>.
Otherwise eslint/prettier will maybe complain.

Reply:
You're right, I forgot. Thank you!